junit5/src/main/java/org/jboss/weld/junit5/auto/TestExtension.java

```diff
@@ -0,0 +1,65 @@
+package org.jboss.weld.junit5.auto;
+
+import java.util.Set;
+
+import javax.enterprise.context.spi.CreationalContext;
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.spi.Extension;
+import javax.enterprise.inject.spi.InjectionPoint;
+import javax.enterprise.inject.spi.InjectionTarget;
+import javax.enterprise.inject.spi.ProcessInjectionTarget;
+
+
+
+public class TestExtension implements Extension {
```

Comment:
We need to come up with better name for this extension.
The only thing I can think of now is TestInstanceWrapper but that's not good either.

Reply:
I chose `TestInstanceInjectionExtension` as that is exactly what it is for.  I've alos